_docs/technical/2017-01-04-updater.md

```diff
@@ -52,14 +52,15 @@ type
     increase the block size limit more than twofold compared to the previous
     limit.
 
+  * `bvdMaxTxSize` – transaction size limit
```

Comment:
Since this document is in `technical` part, I think it's worth noticing that this value limits size of `TxAux` (which is `(Tx, TxWitness, TxDistribution)`).